ct/firefly.sh

```diff
@@ -42,13 +42,13 @@ check_container_resources
     wget -q "https://github.com/firefly-iii/firefly-iii/releases/download/v${RELEASE}/FireflyIII-v${RELEASE}.tar.gz"
     tar -xzf FireflyIII-v${RELEASE}.tar.gz -C /opt/firefly --exclude='storage'
     cd /opt/firefly 
-    composer install --no-dev --no-interaction &>/dev/null
+    #composer install --no-dev --no-interaction &>/dev/null
```

Comment:
While you are at it, the storage folder is never copied back after being copied and wiped out 6 lines above, causing other issues
```suggestion
    cp -r /opt/storage /opt/firefly/storage
    #composer install --no-dev --no-interaction &>/dev/null
```

Reply:
> While you are at it, the storage folder is never copied back after being copied and wiped out 6 lines above, causing other issues
> ```suggestion
>     cp -r /opt/storage /opt/firefly/storage
>     #composer install --no-dev --no-interaction &>/dev/null
> ```

Will be never removed. Its exluded in tarfile